biobesu/suite/hpo_generank/README.md

```diff
@@ -0,0 +1,31 @@
+# hpo_generank
+## General
+The `benchmark_data.tsv` input file is expected to have the following format:
+```
+id  gene_symbol    hpo_ids
+01  A1BG    HP:0001234,HP:0004321
+02  A1CF    HP:0000888
+```
+
+## Lirical
+
+1. [Download](lirical_download) lirical (tested with v1.3.0).
+
+2. Run `java -jar LIRICAL.jar download` (see [here][lirical_prepare] for more information).
+
+3. Run lirical:
+
+   ```bash
+   biobesu hpo_generank lirical --jar /path/to/lirical-1.3.0/LIRICAL.jar \
+   --hpo /path/to/hp.obo --input /path/to/benchmark_data.tsv \
+   --output /path/to/dir/output --lirical_data /path/to/dir/lirical/data \
+   --runner_data /path/to/tmp/dir/
+   ```
+
+Note: `--runner_data` is needed for designating a location where the runner can download temporary data to. When executing the runner multiple times, using the same path skips re-downloading the same data every time.
+
+
+
+
+[lirical_download]: https://github.com/TheJacksonLaboratory/LIRICAL/releases/tag/v1.3.0
```

Comment:
dead link

Reply:
It seems to work on my side. Could you check again?

```
$ curl --head https://github.com/TheJacksonLaboratory/LIRICAL/releases/tag/v1.3.0
HTTP/2 200
```

Reply:
On second thought, you probably refer to [this typo](https://github.com/molgenis/biobesu/pull/1/files/15a5bc08060ef14899be6a41490a207fc4c13816..67b42d785c9f37dbe58efa2c9842561a673ca5e2#diff-159a76fa3d51da3ee19c226428e7cac7f403e3c8301163edfd77cc572a6e0536L12) which has now been fixed. So in that case this issue can be resolved 🙂

Reply:
It was that typo, so It's resolved now